airbyte-integrations/connectors/source-mailchimp/acceptance-test-config.yml

```diff
@@ -12,8 +12,6 @@ acceptance_tests:
         # for auth with oauth2 token
       - config_path: "secrets/config_oauth.json"
         status: "succeed"
-      - config_path: "integration_tests/invalid_config.json"
```

Comment:
Why remove this?

Reply:
it is not relevant, and causes error in selectiveauthenticator